src/components/i18n/locales/en/messages.json

```diff
@@ -19,11 +19,27 @@
   },
   "toolNames": {
     "Text": "",
+    "Heading": "",
```

Comment:
we cant hardcode tools list in the editor core

Reply:
Removed